include/mgp.hpp

```diff
@@ -4340,6 +4340,17 @@ inline List ListAllLabelPropertyIndices(mgp_graph *memgraph_graph) {
   return List(label_property_indices);
 }
 
+inline List SearchTextIndex(mgp_graph *memgraph_graph, std::string_view index_name, std::string_view search_query) {
+  auto results_or_error =
+      Map(mgp::MemHandlerCallback(graph_search_text_index, memgraph_graph, index_name.data(), search_query.data()));
+  auto maybe_error = results_or_error["error_msg"].ValueString();
+  if (!maybe_error.empty()) {
+    throw std::runtime_error{maybe_error.data()};
+  }
+
+  return results_or_error["search_results"].ValueList();
```

Comment:
Do we need some checks here?
What happens if the "search_results" doesn't exist?

Reply:
The creation of `results_or_error` is done in `WrapTextSearch`, which always creates a map with this field (or throws an exception). I can add a check here anyway, just to make sure.